src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -119,16 +122,29 @@ private String getCloudInputs() {
         };
     }
 
+    @Test(dataProvider = "BQSRTest")
+    public void testBQSRSpark(BQSRTest params) throws IOException {
+        ArgumentsBuilder ab = new ArgumentsBuilder().add(params.getCommandLineNoApiKey());
+        IntegrationTestSpec spec = new IntegrationTestSpec(
+                ab.getString(),
+                Arrays.asList(params.expectedFileName));
+        spec.executeTest("testBQSRSpark-" + params.args, this);
+    }
+
+    //This data provider is for tests that use reference (but not BAM) files stored in buckets
     @DataProvider(name = "BQSRCloudTest")
     public Object[][] createBQSRCloudTestData() {
         final String localResources =  getResourceDir();
 
         final String GRCh37RefCloud = ReferenceAPISource.URL_PREFIX + ReferenceAPISource.GRCH37_REF_ID;
+        final String chr2021Reference2bit = GCS_b37_CHR20_21_REFERENCE_2BIT;
         final String hiSeqBam_chr20 = localResources + WGS_B37_CH20_1M_1M1K_BAM;
         final String hiSeqBam_1read = localResources + "overlappingRead.bam";
         final String dbSNPb37_chr20 = localResources + DBSNP_138_B37_CH20_1M_1M1K_VCF;
         final String dbSNPb37_chr2021 = dbsnp_138_b37_20_21_vcf;
 
+        final String HiSeqBamCloud_chr20 = getCloudInputs() + WGS_B37_CH20_1M_1M1K_BAM;
```

Comment:
\[FindBugs\] INFO: DLS: Dead store to HiSeqBamCloud\_chr20 in org\.broadinstitute\.hellbender\.tools\.spark\.BaseRecalibratorSparkIntegrationTest\.createBQSRCloudTestData\(\)